src/contentScript/messenger/api.ts

```diff
@@ -71,6 +71,7 @@ export const setPageState = getMethod("SET_PAGE_STATE");
 
 export const notify = {
   info: getNotifier("NOTIFY_INFO"),
+  // TODO: Automatically report from api.ts because of https://github.com/pixiebrix/pixiebrix-extension/blob/dce0d5cbb54d5fc1a61d720e43d17383a152df2e/src/background/contextMenus.ts#L92-L95
   error: getNotifier("NOTIFY_ERROR"),
```

Comment:
Link: https://github.com/pixiebrix/pixiebrix-extension/blob/main/src/background/contextMenus.ts#L92-L95

I'm thinking about doing something like this, but not a priority since it's used once so far (and it's not this easy due to the types)

```suggestion
  error: (details) => {
    if (details.reportError) {
      reportError(details)
      details.reportError = false
    }
    
    getNotifier("NOTIFY_ERROR")(details)
  }
```

Reply:
Agree it's not an immediate priority